### PR TITLE
Conditionally enable probes measuring in-cluster network latency

### PR DESCRIPTION
**Pull request by oxddr**
ref https://github.com/kubernetes/perf-tests/issues/597

Add couple of minor improvements to the probes framework. 

/assign @mm4tt

### CODE REVIEW

**Comment by oxddr** [comment]
ping

File: clusterloader2/pkg/measurement/common/probes/probes.go [hunk 1]
```diff
@@ -185,17 +186,26 @@ func (p *probesMeasurement) gather(params map[string]interface{}) ([]measurement
 			latency := time.Duration(float64(sample.Value) * float64(time.Second))
 			latencyMetric.SetQuantile(quantile, latency)
 		}
+		prefix := ""
```
Comment (mm4tt) [inline clusterloader2/pkg/measurement/common/probes/probes.go]: NIT: prefix, suffix := "", ""

File: clusterloader2/testing/density/config.yaml [hunk 2]
```diff
@@ -53,6 +54,13 @@ steps:
     Method: APIResponsivenessPrometheus
     Params:
       action: start
+  {{if $ENABLE_PROBES}}
+  - Identifier: Probes
+    Method: Probes
+    Params:
+      action: start
+      replicasPerProbe: {{DivideInt .Nodes 100}}
```
Comment (mm4tt) [inline clusterloader2/testing/density/config.yaml]: As discussed, please add a TODO to re-assess after some time.

**Pull request merged**